src/app/employee/employee-form/employee-form.component.html

```diff
@@ -1,19 +1,19 @@
 <mat-vertical-stepper [linear]="true">
   <mat-step [stepControl]="formGroup.get('bio')">
       <ng-template matStepLabel>Biographical Info</ng-template>
-      <tcp-bio-form (formReady)="registerForm('bio', $event)"></tcp-bio-form>
+      <tcp-bio-form [bio]="(employee$ | async).bio" (formReady)="registerForm('bio', $event)"></tcp-bio-form>
```

Comment:
Should we add `?` to safe check each of the `(employee$ | async)` references in this file? So: `[bio]="(employee$ | async)?.bio"`

Reply:
Yep, I'll add that safety mechanism - good idea!